src/components/alert/alert-component.ts

```diff
@@ -322,7 +322,9 @@ export class AlertCmp {
     this.d.inputs.forEach(i => {
       values[i.name] = i.value;
     });
-    return values;
+
```

Comment:
could we implement this with an early return instead?

```
if (this.d.inputs.length === 0) {
  return undefined;
}
```
this way we don't allocate `values` and Object.keys()'s array.